handler/push.go

```diff
@@ -207,3 +207,12 @@ func splitLabels(labels string) (map[string]string, error) {
 	}
 	return result, nil
 }
+
+// TODO: remove this, Don't know the reason but seems that route.Params is Not working
+// Subtitute for route.Params
+func urlParams(ctx context.Context, p string) string {
```

Comment:
@beorn7 Don't Know the reason but it seems that `route.Params` is not working in this situation. Add manual function that does the same thing.

Reply:
It is correct I just implemented this in the wrong way